templates/vue-starter-template/app/components/account/ActionLink.vue

```diff
@@ -0,0 +1,22 @@
+<script setup lang="ts">
+defineSlots<{
+  default(): unknown;
+}>();
+
+defineProps<{
+  disabled?: boolean;
+}>();
+</script>
+
+<template>
+  <FormLinkButton
```

Comment:
is it a link or button? what action will be invoked when clicking on it?
Combining button and link will break the accessibility

Reply:
It is for buttons that look like a link. Determined by the designs